example/src/client.ts

```diff
@@ -57,9 +54,14 @@ listen({
         // create and start the language client
         const languageClient = createLanguageClient(connection);
         languageClient.onReady().then(() => {
-            languageClient.onRequest(TypeScriptRenameRequest.type, params => {
```

Comment:
I didn't want to remove this, but because of the different language server protocol types it's now failing type checking.